### PR TITLE
chore(driving-license): Load-testing drivers's license components

### PR DESCRIPTION
**Pull request by pshomov**
# ...

We are simply using the feature-deployment

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/api/domains/license-service/src/lib/licenseService.service.ts [hunk 1]
```diff
@@ -237,6 +237,9 @@ export class LicenseServiceService {
     licenseType: GenericLicenseType,
     data: string,
   ): Promise<PkPassVerification> {
+    if (nationalId !== process.env.HI_I_AM_DAVID) {
```
Comment (dabbeg) [inline libs/api/domains/license-service/src/lib/licenseService.service.ts:240]: :grin:

File: infra/src/feature-env.ts [hunk 2]
```diff
@@ -147,6 +151,13 @@ yargs(hideBin(process.argv))
       description:
         'List of comma separated Docker image names that have changed',
     },
+    env: {
+      type: 'string',
+      choices: ['dev', 'staging', 'prod'],
+      default: 'dev',
+      description:
+        'List of comma separated Docker image names that have changed',
```
Comment (sindrig) [inline infra/src/feature-env.ts:159]: fix description

File: libs/api/domains/license-service/src/lib/licenseService.service.ts (hunk 1 above)
Comment (sindrig) [reply libs/api/domains/license-service/src/lib/licenseService.service.ts:241]: HI DAVID I'M ERROR

**Review by sindrig — CHANGES_REQUESTED** [review]

File: libs/api/domains/license-service/src/lib/licenseService.service.ts (hunk 1 above)
Comment (dabbeg) [reply libs/api/domains/license-service/src/lib/licenseService.service.ts:241]: HI SINDRI R U ERROR 2?

**Review by Valur — APPROVED** [review]

**Pull request closed without merging**